src/components/TaskCard/TaskCard.scss

```
@@ -30,4 +30,9 @@
         padding: 0.5rem;
         background-color: base.$bg-gray-50;
     }
+
+    &--complete &__title {
+        text-decoration: line-through;
+        color: base.$color-success;
```

try to write css code like this
```suggestion
        color: $color-success;
```